js/bybit.js

```diff
@@ -2623,7 +2761,7 @@ module.exports = class bybit extends Exchange {
         } else if (lastTradeTimestamp === undefined) {
             lastTradeTimestamp = this.parse8601 (this.safeStringN (order, [ 'updated_time', 'updated_at', 'update_time' ]));
             if (lastTradeTimestamp === undefined) {
-                lastTradeTimestamp = this.safeNumber (order, 'updateTime');
+                lastTradeTimestamp = this.safeNumber (order, 'updateTime'); // todo: trade udpate time is different than update order time
```

Comment:
in that case should we remove it?

Reply:
I wouldnt touch that in this current PR, as it's a different aspect, whether exchange's API engine emits 'updateTime' changes whenever there is trade too, so it might be right in some cases but might be wrong in some cases. however, I'll not touch it in this PR. just made a note for future edits.